tests/unit/format_test.py

```diff
@@ -0,0 +1,244 @@
+import unittest
+
+import dash_table.Format as f
+from dash_table.Format import Format
+import dash_table.FormatTemplate as FormatTemplate
+
+class FormatTest(unittest.TestCase):
+    def validate_complex(self, res):
+        self.assertEqual(res['locale']['symbol'][0], 'a')
+        self.assertEqual(res['locale']['symbol'][1], 'bc')
+        self.assertEqual(res['locale']['decimal'], 'x')
+        self.assertEqual(res['locale']['group'], 'y')
+        self.assertEqual(res['nully'], 'N/A')
+        self.assertEqual(res['prefix'], None)
+        self.assertEqual(res['specifier'], '.^($010,.6s')
+
+    def test_complex_and_valid_in_ctor(self):
+        res = Format(
+            align=f.Align.center,
+            fill='.',
+            group=f.Group.yes,
+            padding=True,
+            padding_width=10,
+            precision=6,
+            scheme='s',
+            sign=f.Sign.parantheses,
+            symbol=f.Symbol.yes,
+            symbol_prefix='a',
+            symbol_suffix='bc',
+            decimal_delimitor='x',
+            group_delimitor='y',
+            groups=[2, 2, 2, 3],
+            nully='N/A',
+            si_prefix=f.Prefix.none
+        )
+
+        self.validate_complex(res.to_plotly_json())
+
+    def test_complex_and_valid_in_fluent(self):
+        res = Format().align(f.Align.center).fill('.').group(f.Group.yes).padding(True).padding_width(10).precision(6).scheme('s').sign(f.Sign.parantheses).symbol(f.Symbol.yes).symbol_prefix('a').symbol_suffix('bc').decimal_delimitor('x').group_delimitor('y').groups([2, 2, 2, 3]).nully('N/A').si_prefix(f.Prefix.none)
+
+        self.validate_complex(res.to_plotly_json())
+
+    def test_money_template(self):
+        res = FormatTemplate.money(2).to_plotly_json()
+
+        self.assertEqual(res['specifier'], '$,.2f')
+
+    def test_percentage_template(self):
+        res = FormatTemplate.percentage(1).to_plotly_json()
+
+        self.assertEqual(res['specifier'], '.1%')
+
+    def test_valid_align_named(self):
+        Format().align(f.Align.center)
+
+    def test_valid_align_string(self):
+        Format().align('=')
+
+    def test_invalid_align_string(self):
+        self.assertRaises(Exception, Format().align, 'i')
```

Comment:
Make sure to update these exception types to match the changed types in source. They'll pass with the base `Exception` but in principle the error could have come from some other problem.